pyproject.toml

```diff
@@ -94,6 +95,10 @@ dev = [
     "datacompy[build]",
 ]
 
+mypy = [
```

Comment:
We can just add this to `QA` extras as we will want stubs installed every time mypy is installed.